Documentation/ceph-filesystem-crd.md

```diff
@@ -9,7 +9,9 @@ indent: true
 Rook allows creation and customization of shared file systems through the custom resource definitions (CRDs). The following settings are available
 for Ceph file systems.
 
-## Sample
+## Samples
+
+### Replicated
```

Comment:
looks like the replicated sample is using erasure coding.

Reply:
Fixed.